stable/presto/templates/configmap-worker.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if gt .Values.server.workers 0.0 }}
```

Comment:
You might want to try the `int` function. 

`{{- if gt (int .Values.server.workers) 0 }}`